packages/nbdime/src/diff/widget/notebook.ts

```diff
@@ -1,45 +1,45 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
-'use strict';
+"use strict";
 
-import {
-  Panel
-} from '@lumino/widgets';
+import { Panel } from "@lumino/widgets";
 
 import type {
   IRenderMimeRegistry
 } from '@jupyterlab/rendermime';
 
+import { LazyDisplayLinkedListCell, LinkedListCell } from "./linked-cells";
+import { CellDiffWidget } from "./cell";
 import {
-  CellDiffWidget
-} from './cell';
+  CHUNK_PANEL_CLASS,
+  ADDED_CHUNK_PANEL_CLASS,
+  REMOVED_CHUNK_PANEL_CLASS,
+} from "./common";
 
-import {
-  CHUNK_PANEL_CLASS, ADDED_CHUNK_PANEL_CLASS, REMOVED_CHUNK_PANEL_CLASS
-} from './common';
+import { MetadataDiffWidget } from "./metadata";
 
-import {
-  MetadataDiffWidget
-} from './metadata';
+import type { NotebookDiffModel, CellDiffModel } from "../model";
 
-import type {
-  NotebookDiffModel
-} from '../model';
-
-
-const NBDIFF_CLASS = 'jp-Notebook-diff';
+const NBDIFF_CLASS = "jp-Notebook-diff";
 
 
 /**
  * NotebookDiffWidget
  */
-export
-class NotebookDiffWidget extends Panel {
-  constructor(model: NotebookDiffModel, rendermime: IRenderMimeRegistry) {
+export class NotebookDiffWidget extends Panel {
+  private _model: NotebookDiffModel;
+  private _rendermime: IRenderMimeRegistry;
+  private previousCell: LinkedListCell | null;
```

Comment:
The previous layout follows the JupyterLab style of having private declarations on the bottom. I think there is value in having consistency in style with lab.